launchdarkly-android-client-sdk/src/main/java/com/launchdarkly/sdk/android/ContextDataManager.java

```diff
@@ -122,26 +115,6 @@ public void initData(
         initDataInternal(context, newData, true);
     }
 
-    /**
-     * Attempts to initialize the flag data state from the persistent store. If there was a set of
-     * stored flag data for this context, it updates the current flag state and returns true; also,
-     * the context is added to the list of stored contexts-- evicting old context data if necessary.
-     * If there was no stored data, it leaves the current flag state as is and returns false.
-     *
-     * @param context the new context
-     * @return true if successful
-     */
-    public boolean initFromStoredData(@NonNull LDContext context) {
```

Comment:
For reviewers:  This functionality was moved to switchToContext.  This also helps to privatize the concept of stored data so that other components don't have to be involved.  Yay separation of concerns.